db/db_impl/replication_codec.cc

```diff
@@ -32,4 +35,27 @@ std::string RecordMemTableSwitch(
   SerializeMemTableSwitchRecord(&rlr.contents, record);
   return replication_log_listener->OnReplicationLogRecord(std::move(rlr));
 }
+
+Status SerializeCloudManifestDelta(std::string* dst, CloudManifestDelta delta) {
+  PutVarint64(dst, delta.file_num);
```

Comment:
Maybe add a version to this, just like `MemtableSwitch` above?

Edit: Although, see above, we might not need to serialize the cloud manifest delta in RocksDB codebase.

Reply:
actually, I'm thinking about removing the version in `MemtableSwitch`. Seems it's not necessary. If we introduce some backwards incompatible change, we can always introduce a new event type(e.g., `kMemtableSwitch2` )